web/themes/custom/bevan/package-lock.json

```diff
@@ -99,8 +99,8 @@
       "integrity": "sha512-5hYdAkZlcG8tOLujVDTgCT+uPX0VnpAH28gWsLfzpXYm7wP6mp5Q/gYyR7YQ0cKVJcXJnl3j2kpBan13PtQf6w==",
       "dev": true,
       "requires": {
-        "delegates": "1.0.0",
-        "readable-stream": "2.3.6"
+        "delegates": "^1.0.0",
+        "readable-stream": "^2.0.6"
```

Comment:
There's quite a lot of changes in `package-lock.json` including some odd looking requirements of above-a-much-lower-version for a package. Are these all kosher?

Reply:
@joesb - I haven't touched that so I'm a little confused on this to be honest, I don't understand why the project install changed the `settings.php` and this `.json` file, when they're not supposed to be.

Reply:
Ok, no probs. I'll take a look. There's a some merge conflicts now including in `styles.css`, so maybe we can revert this file and rebuild the `styles.css` file to handle merge conflicts in the stylesheets.